bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -793,6 +781,63 @@ const Whiteboard = React.memo(function Whiteboard(props) {
     presentationAreaHeight,
   ]);
 
+  React.useEffect (() => {
+    const bbb_multi_user_tools = getFromUserSettings(
+      'bbb_multi_user_tools',
+      meetingClientSettingsInitialValues.public.whiteboard.toolbar.multiUserTools,
+    );
+      const allElements = document.querySelectorAll('[data-testid^="tools."]');
+    
+      if (bbb_multi_user_tools.length >= 1 && !isModerator) {
+        allElements.forEach((element) => {
+          const toolName = element.getAttribute('data-testid').split('.')[1];
+    
+          if (!bbb_multi_user_tools.includes(toolName)) {
+            element.style.display = 'none';

```

Comment:
```suggestion
            element.style?.display = 'none';
```

being overly cautions